library/strings/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'com.likethesalad.stem-library'
```

Comment:
This is what was blocking from doing that work before. Are the string with template (such as ${app_name}) still working as expected?

Reply:
(OK covered by your test, sorry)